Commander.Cli/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Cqse.Teamscale.Profiler.Commons.Ipc;
+using System.Reflection;
+
+var profilerIpc = new ProfilerIpc(new IpcConfig());
+
+while(true)
+{
+    Command nextCommand = Command.Start;
+    if (!string.IsNullOrEmpty(profilerIpc.TestName))
+    {
+        nextCommand = Command.Stop;
+    }
+
+    (Command command, string? arg) = WaitForCommand(nextCommand);
+    if (ExecuteCommand(command, arg))
+    {
+        return;
+    }
+}
+
+bool ExecuteCommand(Command command, string? arg)
+{
+    switch(command)
+    {
+        case Command.Start:
+            profilerIpc.StartTest(arg);
+            Help($"Started test {arg}");
+            return false;
+        case Command.Stop:
+            if (arg?.Length > 1)
+            {
+                arg = char.ToUpper(arg[0]) + arg.Substring(1).ToLower();
+                if (Enum.TryParse(arg, out TestExecutionResult result))
```

Comment:
You could use the [ignorecase varian](https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-6.0#system-enum-tryparse(system-type-system-readonlyspan((system-char))-system-boolean-system-object@))t of TryParse

Reply:
:facepalm: